src/h5web/dimension-mapper/AxisMapper.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
   axis: Axis;
   rawDims: number[];
   mapperState: NonNullable<DimensionMapping>;
-  onChange(mapperState: DimensionMapping): void;
+  onChange: (mapperState: DimensionMapping) => void;
```

Comment:
With `onChange` declared as a method, ESLint wants us to call it with `props.onChange(...)` instead of just `onChange(...)` to ensure that `this` is properly bound. Of course we don't care about `this` here, so a simple function is more adequate.